backend/tests/test_preprocess.py

```diff
@@ -1,14 +1,40 @@
 import unittest
 from backend.services.preprocess import Preprocessor
+from backend.services.preprocess_bug_report import preprocess_bug_report
 
 class test_preprocess(unittest.TestCase):
     def test_wrong_path(self):
         text = "uhhh lets preProcess some texts!!"
-        stop_words_path = "/stop_words/java-keywords-bugs.txt"
+        stop_words_path = "stop_words/java-keywords-bugs.txt"
 
         expected_result = ""
         actual_result = Preprocessor.preprocess_text(text, stop_words_path)
 
         self.assertEqual(expected_result, actual_result)
+    
+    def test_bug_report_preprocessing(self):
+        self.maxDiff = None
 
-    
\ No newline at end of file
+        bug_report_path = "temp_testing/bug_report_10.txt"
```

Comment:
Not sure if you forgot to update this path, but I'm only seeing `/temp_testing/bug_report_129.txt` in this PR.